pkg/util/maven/maven_command.go

```diff
@@ -74,6 +74,17 @@ func (c *Command) Do(ctx context.Context) error {
 		args = append(args, "--settings", settingsPath)
 	}
 
+	if GOMAXPROCS, ok := os.LookupEnv("GOMAXPROCS"); ok {
+		args = append(args, "-Dmaven.artifact.threads="+GOMAXPROCS)
```

Comment:
It should be checked that the `maven.artifact.threads` option is not already provided in `AdditionalArguments`, to avoid overriding user provided value.